README.md

```diff
@@ -33,7 +33,7 @@ This project is a complete chess platform, allowing users to play chess games, a
 ## Building and Running
 There are two binary packages that can be compiled and ran: `client` and `server`. Execute `cargo [build | run]` with the `-p` (package) flag, followed by the package:
```

Comment:
I guess technically this whole section is wrong now, since there's no server package

Reply:
made issue